src/main/groovy/io/github/cosmicsilence/compat/GradleCompat.groovy

```diff
@@ -25,4 +26,18 @@ abstract class GradleCompat {
 
         return fileProp
     }
+
+    static Property<Boolean> booleanProperty(Project project, Boolean defaultBoolean = null) {
+        def booleanProp = project.objects.property(Boolean)
+
+        if (defaultBoolean) {
+            if (isVersion4(project)) {
+                booleanProp.set(project.provider { defaultBoolean })
```

Comment:
`(` instead of `{` here?

Reply:
It has to be `{` as it is defining a closure.